### DISCUSSION TITLE
New idea for addon!

### DISCUSSION THREAD

**Discussion by TheFirstOrangeScratcher7777** [ScratchAddons/ScratchAddons#5843] [Ideas] [answered]
Can Scartch Addons have a right and left shift extra keys? I asking because I need it do to a supe rrpuper cool game where is speed run!

**Comment by CST1229** [comment, marked as answer]
This wouldn't be compatible with normal Scratch (not even by JSON hacking; the Scratch runtime literally *ignores* those keys). However, [TurboWarp](https://turbowarp.org) does support those keys (it has an extra option in the Extra key options addon called "Show TurboWarp keys" that shows keys like Shift, Control and Escape in the dropdown), so if you're okay with your game only working in TurboWarp, then you can use that.

**Reply by TheFirstOrangeScratcher7777** [reply]
oh, ok, but, I think taht look so fun,  oh, yep, everyone use hacks to do shift key, i do not like hacks, I think everybody do not like hacks, so, can it be?

**Reply by TheFirstOrangeScratcher7777** [reply]
😥, can you do that, pls, pls, pls, pls, pls, pls, pls!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
